### PR TITLE
added creating kondo for provider

### PR DESCRIPTION
**Pull request by hiroya210**
I added function for Provider to create kondos, needed to add Default value to `service_duration `because it throws  `a not-null constraint` error in migration

the form is needs more polishing, but the basic functionality works

### CODE REVIEW

File: app/policies/application_policy.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def show?
   end
 
   def create?
-    false
+    true
```
Comment (spioupiou) [inline app/policies/application_policy.rb:20]: I think we're not supposed to change the application_policy file, only the kondo_policy

Comment (hiroya210) [reply app/policies/application_policy.rb:20]: I'll leave that to you 🤣 I needed temporarily change that to test if create kondo works 😁

Comment (spioupiou) [reply app/policies/application_policy.rb:20]: Lol it's strange, I thought the pundit would stop once it finds the create method in kondo_policy... Then it means application_policy overwrite everything? haha I am confused now

def create?
  current_user.provider?
end

Comment (hiroya210) [reply app/policies/application_policy.rb:20]: I was actually wondering that myself, and then again I didnt want to spend a lot of time to debug it so I just temporarily set it to true haha

Comment (spioupiou) [reply app/policies/application_policy.rb:20]: So weird 😂😂 anyway I’ll merge your branch, we can always set it back to false later!

**Pull request merged**